CHANGELOG.rst

```diff
@@ -12,6 +13,21 @@ This document describes changes between each past release.
   ``cliquet.storage_pool_maxconn`` and ``cliquet.basic_auth_enabled``
   were removed (ref #448)
```

Comment:
Maybe add a mention about what happened on Syncto : prefixed settings won't work if `project_name` is not set (either from `cliquet.initialize()` or using `cliquet.project_name` setting).

Mostly breaking : settings should be read without prefix in code (`request.registry.settings['max_duration']` instead of `request.registry.settings['cliquet.max_duration']`)